src/core/hooks.ts

```diff
@@ -1,74 +1,86 @@
+import { fileURLToPath } from 'url'
 import { resolve, posix } from 'path'
-import merge from 'lodash.mergewith'
+import type { Nuxt } from '@nuxt/schema'
+import type { Consola } from 'consola'
+import { defu } from 'defu'
+import { addPluginTemplate, addTemplate } from '@nuxt/kit'
+import type { Configuration as WebpackConfig } from 'webpack'
+import type { SentryCliPluginOptions } from '@sentry/webpack-plugin'
+import type { Options } from '@sentry/types'
 import * as Sentry from '@sentry/node'
-import { canInitialize, clientSentryEnabled, envToBool, serverSentryEnabled } from './utils'
-import { resolveRelease, resolveClientOptions, resolveServerOptions } from './options'
+import type { ModuleConfiguration } from '../../types'
+import { clientSentryEnabled, serverSentryEnabled, envToBool, canInitialize } from './utils'
+import { resolveRelease, resolvedClientOptions, resolveClientOptions, resolvedServerOptions, resolveServerOptions } from './options'
 
 const RESOLVED_RELEASE_FILENAME = 'sentry.release.config.js'
 
 /**
  * Resolves the options and creates the plugins and the templates at build time.
  *
- * @param      {ThisParameterType<import('@nuxt/types').Module>} moduleContainer
- * @param      {import('../../types/sentry').ResolvedModuleConfiguration} moduleOptions The module options
- * @param      {import('consola').Consola} logger The logger
+ * @param      {Nuxt} nuxt
+ * @param      {ModuleConfiguration} moduleOptions The module options
+ * @param      {Consola} logger The logger
  * @return     {Promise<void>}
```

Comment:
We don't need those JSDoc annotations anymore when specifying types inline. The extra descriptions don't provide any extra value anyway.

Reply:
Removed in [`b01023c`](https://github.com/nuxt-community/sentry-module/pull/456/commits/b01023cc5b7ddcd125fe4287e3ac882e86267fc3)